projects/plugins/wpcomsh/wpcom-features/class-wpcom-features.php

```diff
@@ -1089,12 +1089,8 @@ class WPCOM_Features {
 			),
 		),
 		self::SOCIAL_CONNECTIONS_MANAGEMENT     => array(
-			array(
-				// This feature isn't launched yet, so we're ensuring that it's not available on any plans.
-				'before' => '1900-01-01',
-				self::WPCOM_ALL_SITES,
-				self::JETPACK_ALL_SITES,
-			),
+			self::WPCOM_ALL_SITES,
```

Comment:
I don't know if we should do this as part of this change, we can sync the feature stuff when we have everything ready

Reply:
Reverted.